app/controllers/gsa18f/dashboard_controller.rb

```diff
@@ -1,5 +1,7 @@
 module Gsa18f
   class DashboardController < ApplicationController
+    before_action :check_disabled_client
```

Comment:
seems like this `check_disabled_client` check should be in `application_controller` and just selectively skipped? (I think the only page we want disabled clients to be able to see is the home page, right?) That would ensure that future controllers also use the check rather than needing to explicitly add it to each controller